app/controllers/dashboard.js

```diff
@@ -0,0 +1,28 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { use } from 'ember-could-get-used-to-this';
+import AsyncProcess from 'ilios-common/classes/async-process';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class DashboardController extends Controller {
```

Comment:
There aren't any reports on the LTI dashboard, so most of this isn't needed (maybe none of it).

Reply:
:heavy_check_mark: